### DISCUSSION TITLE
Population PbPk model

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#1032] [Questions & Problems]
I have been working on a population pbpk model using pksim for the medicine Rivaroxaban, trying to simulate a multiple dose oral model and i have been having some issues regarding the <Nan> error when calculating Cl,Vss and other pk related data.(as shown in the attachments)
I could really use your help solving this problem![20211126_181314.jpg](https://user-images.githubusercontent.com/90548695/143884159-f340bb8a-07e6-490a-bffc-d0256e9b44b8.jpg)

**Comment by tobiasK2001** [comment]
Hi,

these values are available only after single dosing. If you think that is not enough, you might place a feature request for that.

Best, Tobias

**Reply by ranamjz** [reply]
Do you mean only single dosing of an individual ?

**Comment by ranamjz** [comment]
Thank you very much sir, 
But the problem is that even after running a population with a single dose , the values are still not available for said population i can only have values after running an individual simulation .

**Comment by tobiasK2001** [comment]
Ah, but maybe there is another way:
create a Box Whisker analysis in the "Run & Analyse Tab" and the Wizard opens
![image](https://user-images.githubusercontent.com/25585248/145383494-c9d20c34-35ef-43a0-9e83-7fd539418a4c.png)
 There you can select the PK-Parameters in question (and give them some sensible units):
![image](https://user-images.githubusercontent.com/25585248/145383702-cf1610b0-1caa-4bb5-9a7a-bf93930dd706.png)

check the outputs you want to see
![image](https://user-images.githubusercontent.com/25585248/145384029-048a8986-9b74-467b-aa1f-d0856017d627.png)

You can get the exact values for Median Individuals or respective percentile either by mouse over tool tip
![image](https://user-images.githubusercontent.com/25585248/145386001-eb6a0b31-7974-4511-aec1-197e554c61ac.png)

 or you can export the results to an excel file by right click on the Box Whisker Analysis and chose Export to excel from the Submenu

@msevestre, @Yuri05 : It is supposed to be that way I guess, but it is not the user friendliest way to unravel that information. It would be nicer to see the values directly with the PK-Analysis. Or a hint that this values can be found somewhere else instead of just NaN. What do you think?
